source/common/types/cardano-node.types.js

```diff
@@ -83,9 +83,9 @@ export const CardanoProcessNameOptions: {
   linux: CardanoNodeProcessNames,
   darwin: CardanoNodeProcessNames,
 } = {
-  win32: 'cardano-node.exe',
-  linux: 'cardano-node',
-  darwin: 'cardano-node',
+  win32: 'jormungandr.exe',
+  linux: 'jormungandr',
+  darwin: 'jormungandr',
```

Comment:
These need to be `cardano-node-jormungandr[.exe]`.

Reply:
@rvl this has been fixed. Can you please review again?